Tests/Transmute/ModuleDependencyTests.swift

```diff
@@ -12,128 +12,147 @@
 import class PackageType.Module
 import XCTest
 
+func testModules(file: StaticString = #file, line: UInt = #line, body: () throws -> Void) {
```

Comment:
For future reference, you can just change XCTestCase functions to have `throws` now.